framework/src/modules/chain/chain.js

```diff
@@ -330,7 +330,7 @@ module.exports = class Chain {
 					action.params.ids,
 				);
 
-				return commonBlock ? commonBlock.id : null;
+				return commonBlock || null;
```

Comment:
Do we need `null` here?

```suggestion
				return commonBlock;
```